Datatrans.Checkout/Managers/DatatransCheckoutPaymentMethod.cs

```diff
@@ -212,8 +217,10 @@ public override PostProcessPaymentResult PostProcessPayment(PostProcessPaymentEv
             }
             else
             {
+                const int defaultErrorCode = -1000;
```

Comment:
what means this magic number -1000?

Reply:
Name of it, not enough meaningful?

Reply:
Actually no. why -1000? Is there have a description of all error codes anywhere?
I guess should be better if we place these constants anywhere with reference to documentation

Reply:
You suggest place all error codes just only for description? We need a default error code, which does not described in documentation, that's why -1000